### DISCUSSION TITLE
Не передается Feature в clusterer

### DISCUSSION THREAD

**Discussion by Wate835** [yandex-maps-unofficial/vue-yandex-maps#309] [Q&A (2.x)] [answered]
В модуле YandexMapClusterer, в доке написано что
`Если вы посмотрите в документацию Яндекса, там также будут вышеуказанные параметры. В нашей обертке, передавать их не требуется - они передадутся в Яндекса автоматически.`
Но у меня никакой информации там нет внутри кроме координат.
Я так и не нашёл куда можно передать, чтобы в кластере получить эти данные.

**Comment by daniluk4000** [comment]
https://yandex-maps-unofficial.github.io/vue-yandex-maps/components/modules/clusterer.html#cluster

Добрый день! Вам фичи приходят в слоте

**Reply by daniluk4000** [reply]
Также я в теории могу добавить событие с этими данными, но пока никому не требовалось

**Reply by Wate835** [reply]
Я понимаю, но там нет данных объектов

**Reply by daniluk4000** [reply]
Есть, clusterer.features

Или вы хотите сказать, что там пусто?

**Reply by Wate835** [reply]
Там не пусто, но внутри должны быть данные из массива объектов
![image](https://github.com/yandex-maps-unofficial/vue-yandex-maps/assets/97688591/1ad16781-51e1-4591-9ce2-b5f6c043fb62)

**Reply by daniluk4000** [reply]
Не должны быть))) Это фичи Яндекса

В общем, чтобы вам тут получить свои объекты, вам нужно задать им properties - и там указать что угодно другое, что сюда и придет

**Reply by Wate835** [reply]
Где то в этом месте должны быть данные из массива моих объектов

**Reply by daniluk4000** [reply]
Только пропертями, массив объектов в Яндекс не передается за ненадобностью

**Reply by Wate835** [reply]
а где имено указать пропертис надо, подскажите плз

**Reply by daniluk4000** [reply, marked as answer]
В settings у ваших маркеров

Аля вот: https://yandex-maps-unofficial.github.io/vue-yandex-maps/components/modules/hint.html#%D0%BF%D1%80%D0%BE%D1%81%D1%82%D0%B5%D0%B8%D1%88%D0%B8%D0%B8-%D0%BF%D1%80%D0%B8%D0%BC%D0%B5%D1%80

**Reply by daniluk4000** [reply]
И там любой объект

**Reply by daniluk4000** [reply]
Хотя знаете что. Я похоже их не сохраняю *смайлик клоуна*

Заведите пожалуйста feature request, я вам сделаю на этой недельке сохранение пропертей. А то действительно грустно получается

**Reply by Wate835** [reply]
Я потестил, вроде данные в пропертис приходят в кластер

**Reply by Wate835** [reply]
Я ещё потесчу, но кажется спасибо за помощь)

**Reply by daniluk4000** [reply]
Оки, если все ок закрывайте))
